hack/after-build/update-generated-conversions.sh

```diff
@@ -39,13 +39,23 @@ EOF
 
   "${genconversion}" -v "${version}" -f - >>  "$TMPFILE"
 
-	mv "$TMPFILE" "pkg/${version}/conversion_generated.go"
+    #TODO: remove this case if we rename the directory expapi/ to experimental/
+    if [ "${version%/*}" == "experimental" ]; then
+        mv "$TMPFILE" "pkg/expapi/${version##*/}/conversion_generated.go"
+    else
+	    mv "$TMPFILE" "pkg/${version}/conversion_generated.go"
+    fi
 }
 
-DEFAULT_VERSIONS="api/v1 expapi/v1"
```

Comment:
"api/v1" is correct here.

Sorry it looks confusing, it's because we want the code here to be consistent with the code for generating deep copies, where using "api/v1" is sort of inevitable.